extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -71,6 +87,9 @@ export class ConsentStateManager {
 
     /** @private {!Promise} */
     this.hasAllPurposeConsentsPromise_ = allPurposeConsentsDeferred.promise;
+
+    /** @public {?string} */
+    this.consentPageViewId64 = once(() => getRandomString64(this.ampdoc_.win));
```

Comment:
Please update the type annotation.
```suggestion
    /** @public @const {function():string} */
    this.consentPageViewId64 = once(() => getRandomString64(this.ampdoc_.win));
```